### PR TITLE
multipass: doc fix and require check for Multipass._run()

### PR DESCRIPTION
**Pull request by cjp256**
All callers specify check=, but better to require it than
    to change the default.

### CODE REVIEW

File: craft_providers/multipass/multipass.py [hunk 1]
```diff
@@ -50,10 +50,10 @@ def _run(
         self,
         command: List[str],
         *,
-        check=True,
+        check: bool,
         **kwargs,
     ) -> subprocess.CompletedProcess:
-        """Execute command in instance_name, allowing output to console."""
+        """Execute command on host, allowing output to console."""
```
Comment (sergiusens) [inline craft_providers/multipass/multipass.py:56]: I don understand, from the point of view of the user of the API, either that or "host" is too generic.

File: craft_providers/multipass/multipass.py [hunk 2]
```diff
@@ -50,10 +50,10 @@ def _run(
         self,
         command: List[str],
         *,
-        check=True,
+        check: bool,
```
Comment (sergiusens) [inline craft_providers/multipass/multipass.py:53]: I don't mind this change, but the PR should probably mention why

**Review by sergiusens — CHANGES_REQUESTED** [review]

File: craft_providers/multipass/multipass.py (hunk 2 above)
Comment (cjp256) [reply craft_providers/multipass/multipass.py:53]: Agreed, fixed.

File: craft_providers/multipass/multipass.py (hunk 1 above)
Comment (cjp256) [reply craft_providers/multipass/multipass.py]: Fair enough, simplified to `execute multipass command` :D

**Review by sergiusens — APPROVED** [review]

**Review by cmatsuoka — APPROVED** [review]

**Pull request merged**